crypto-test/.js/src/test/scala/org/bitcoins/crypto/KeysTest.scala

```diff
@@ -18,4 +17,62 @@ class KeysTest extends AnyFlatSpec with Matchers {
     assert(!BCryptoCryptoRuntime.isValidPubKey(privkey.bytes))
   }
 
+  it must "be able to compress/decompress public keys" in {
+    val privkey = ECPrivateKey.freshPrivateKey
+    assert(BCryptoCryptoRuntime.secKeyVerify(privkey.bytes))
+    assert(privkey.isCompressed)
+
+    val pubkey = BCryptoCryptoRuntime.toPublicKey(privkey, isCompressed = false)
+    assert(BCryptoCryptoRuntime.isValidPubKey(pubkey.bytes))
+    assert(!pubkey.isCompressed)
+
+    val compressed = privkey.publicKey
+    assert(BCryptoCryptoRuntime.isValidPubKey(compressed.bytes))
+    assert(compressed.isCompressed)
+
+    val converted = ECPublicKey.fromBytes(
+      BCryptoCryptoRuntime.publicKeyConvert(pubkey.bytes, compressed = true))
+    assert(BCryptoCryptoRuntime.isValidPubKey(converted.bytes))
+    assert(converted.isCompressed)
+
+    val decompressed = ECPublicKey.fromBytes(
+      BCryptoCryptoRuntime.publicKeyConvert(compressed.bytes,
+                                            compressed = false))
+    assert(BCryptoCryptoRuntime.isValidPubKey(decompressed.bytes))
+    assert(!decompressed.isCompressed)
+
+    assert(pubkey.bytes != converted.bytes)
+    assert(compressed.bytes == converted.bytes)
+    assert(compressed.bytes != decompressed.bytes)
+    assert(pubkey.bytes == decompressed.bytes)
+  }
+
+  private val inf = ECPublicKey.fromHex("00")
```

Comment:
nit: Can we put this inside of 

```scala 
object ECPublicKey {
  val inf = ECPublicKey.fromHex("00")
} 
```
or if you take my other suggestion 

```scala
object ECPublicKey { 
  val inf = ECPointInfinity.publicKey
}
```

Reply:
nit: if we have a val like this it shouldn't be called `inf` which is short in math for `infimum` not `infinity`